modules/core/04-channel/keeper/packet_test.go

```diff
@@ -611,11 +656,7 @@ func (suite *KeeperTestSuite) TestRecvPacket() {
 				}
 			} else {
 				suite.Require().Error(err)
-
-				// only check if expError is set, since not all error codes can be known
-				if expError != nil {
-					suite.Require().True(errors.Is(err, expError))
-				}
+				suite.Require().True(errors.Is(err, tc.expError))
```

Comment:
nit: I think you can do: `suite.Require().ErrorIs(err, tc.expError)`

Reply:
ugh, yea, that slipped through